README.md

```diff
@@ -5,6 +5,7 @@
 > This is simple web app is designed to provide various types of meal, 
 > to give the users desire meals, user can comment and rating the meals.
 
+### [video link](https://www.loom.com/share/d0089692671b40899f947865b81fff76)

```

Comment:
Optional - in your video presentation, please make you are audible to explain your project better